src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -587,6 +611,19 @@ internal AggregatingLogger InitializeLogger(AnalyzeOptionsBase analyzeOptions)
             if (filePath != null)
             {
                 context.TargetUri = new Uri(filePath);
+
+                if (_computeHashes)
```

Comment:
>_computeHashes [](http://example.com/codeflow?start=20&length=14)

computing once per file and re-using if that already exists. We must update _pathToHashDataMap because it will be used to create the artifacts